django/urls/resolvers.py

```diff
@@ -206,6 +207,9 @@ def _route_to_regex(route, is_endpoint=False):
     For example, 'foo/<int:pk>' returns '^foo\\/(?P<pk>[0-9]+)'
     and {'pk': <django.urls.converters.IntConverter>}.
     """
+    if any((s in route) for s in string.whitespace):
```

Comment:
You want to use set operations here: 

```suggestion
    if not set(route).isdisjoint(string.whitespace):
```